ecommerce/frontend/src/components/UserReview.js

```diff
@@ -0,0 +1,128 @@
+    const [isLoading, setIsLoading] = useState(true)
+    const [currentPage, setCurrentPage] = useState(1)
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        // When current page is changed, makes a call to backend and gets the new reviews.
+        async function fetch() {
+            try {
+                setIsLoading(true)
+                const tmp = [
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
```

Comment:
the `date` coming back from backend is likely going to be Date ISO8601 string not already formatted as `Month 11` or something.

it will be something like `2011-10-05T14:48:00.000Z`. parsing into a Date object (or using momentjs date library) will be required.